packages/tool-cache/__tests__/tool-cache.test.ts

```diff
@@ -198,6 +199,39 @@ describe('@actions/tool-cache', function() {
       }
     })
   } else {
+    if (IS_MAC) {
+      it('extract .xar', async () => {
+        const tempDir = path.join(tempPath, 'test-install.xar')
```

Comment:
Can we use the cli to generate a xar on the fly? 

I'd rather not check in binaries here.

Reply:
Agreed.